tests/cacl/test_cacl_application.py

```diff
@@ -682,11 +682,9 @@ def generate_scale_rules(duthost, ip_type):
     # to call check_iptable_rules every 10s to keep ssh session alive, it just calls
     # duthost.command to active ssh connection.
     # In this way, we can active ssh connection and wait as long as we want.
-    if duthost.is_multi_asic:
-        # For multi-asic, it has to wait enough long
-        wait_until(200, 10, 2, check_iptable_rules, duthost)
-    else:
-        wait_until(30, 10, 2, check_iptable_rules, duthost)
+
+    # For multi-asic, it has to wait enough long
+    wait_until(200, 10, 2, check_iptable_rules, duthost)
```

Comment:
Should we still use different timeout for different testbed? As you mentioned, some t1 lag testbeds take longer time? Due to poor CPU performance or scale is different?

If it is not for multi-asic, maybe we can delete the comments, it is confusing.

Reply:
Originally, I used different timeout for different testbed, but it turns out to be flaky issue.
Not sure the specific reason, but the time of refreshing cacl rules is different, from log, caclmgrd will refresh it once more. It may related to the issue I reported https://github.com/Azure/sonic-buildimage/issues/9824.
But eventually, iptables rules will be added, for saving time to check flaky failures every time, it's better to wait long enough to get stable test result.
I modified the comment as your suggestion.